Azure Sentinel Query Creator.ipynb

```diff
@@ -0,0 +1,1353 @@
+{
```

Comment:
<p>I've started to put identifiers into our entities. We should update them to match this (and the official docs) including adding the missing entities.</p><p> </p><p>I wonder if we should put this (or parts of it) in msticpy - it's kind of cool. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>